core/src/main/scala/libretto/ClosedLib.scala

```diff
@@ -43,23 +43,23 @@ class ClosedLib[
         out(f)
     }
 
-  implicit class ClosedLinearFunctionOps[A, B](self: A -⚬ B) {
+  extension [A, B](ƒ: A -⚬ B) {
```

Comment:
Not sure you like the `ƒ`.

If so, I would use more of it! :grinning:

Reply:
`f` is fine.